nimbus.fml.yaml

```
@@ -70,6 +70,17 @@ features:
             "libraryShortcuts": true
           }
         }
+      - channel: beta
+        value: {
+          "sections-enabled": {
+            "topSites": true,
+            "jumpBackIn": true,
+            "recentlySaved": true,
+            "recentExplorations": true,
+            "pocket": true,
+            "libraryShortcuts": false
```

```suggestion
            "top-sites": true,
            "jump-back-in": true,
            "recently-saved": true,
            "recent-explorations": true,
            "pocket": true,
            "library-shortcuts": false
```

Just made a PR of my own to update the fml back to original values. We should reflect that.